### DISCUSSION TITLE
Animate only when next page is loaded and ready to be displayed

### DISCUSSION THREAD

**Discussion by YoanRouleau** [swup/swup#502] [Help] [answered]
Hi there!

I've been trying Swup which happened to be extremely good, and way easier than (for instance) barbaJS. I'm trying to get the same result as this website : https://www.7h34.fr/projets/aquarium-la-rochelle/. Basically, when you scroll at the bottom end, you can direclty access the next page without delay between "in" and "out": everything looks in sync.
Right now, i'm using the SlideTheme and managed to remove the opacity effect so i'm getting quite the same result, but since there's a delay between page 1 and page 2 (load state I guess), i'm almost "done".

Any ideas ?

**Comment by daun** [comment]
Do you mean having both pages visible at the same time? Or only starting the transition once the page HTML was loaded?

**Reply by YoanRouleau** [reply]
Yeah, that's about being able to see both pages at the same time. :)

**Comment by daun** [comment, marked as answer]
**Update**: the recently released [swup 4](https://swup.js.org/announcements/swup-4/) supports [parallel animations](https://swup.js.org/plugins/parallel-plugin/).

Previous answer: it can be done with some customization by hooking into swup events. See prior issues/discussions for details:

- [Sync option like Barba?](https://github.com/swup/swup/issues/337)
- [Is it possible to start the transition of both pages at the same time?](https://github.com/swup/swup/issues/447)

**Reply by YoanRouleau** [reply]
Alright! Thanks for your time and answers.  :D
